subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,76 @@
+
+#### Comprehension
+
+##### Describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### Was the CI/CD pipeline, integrating Jenkins and SonarQube, effectively used to improve the development process?
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
+
+##### Test each microservice API.
+
+###### Did all microservice APIs function correctly and handle errors effectively?
+
+##### Verify Admin Dashboard functionality.
+
+###### Did the Admin Dashboard correctly manage users, travels, and payment gateways?
```

Comment:
What do we mean by "correctly manage users, travellers and payment gateways"? 
- Which type of actions can we do from the admin dashboard? Retrieve information, approve or change information ...?
This links to the comment I made on the subject: I feel we should be clearer on what the functionalities are expected from the student's final application - once defined in the subject, they should be either checked here one by one or there should be something like: `Did all the `Admin Dashboard` features defined in the subject work as expected?`

Reply:
Admins should be able to perform CRUD operations for users, travels and payment gateways?

Reply:
> Admins should be able to perform CRUD operations for users, travels and payment gateways?

yes

Reply:
Please, add this information in the audit question. It could be nice to add, for example:
```
Admin should be able to perform CRUD operations for users, travels and payment methods.
For each "entity" (user, travel, payment method) try to create, read, update and delete.

Is everything working as expected?
Are errors handled correctly?
```